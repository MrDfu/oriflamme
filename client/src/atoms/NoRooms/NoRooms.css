.no-rooms {
  font-family: var(--font-family-gothic);
  font-size: var(--font-size-biggest);
  text-align: center;
}