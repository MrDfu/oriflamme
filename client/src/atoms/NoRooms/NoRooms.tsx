import React from 'react';
import './NoRooms.css';

export default function NoRooms () {
  return (
    <div className="no-rooms">No rooms at the moment</div>
  );
}