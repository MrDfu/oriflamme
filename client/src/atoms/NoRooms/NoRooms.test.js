import {render, screen} from '@testing-library/react'
import NoRooms from './NoRooms'
import * as React from 'react'


describe('NoRooms component', ()=>{

    test('should match the snapshot',()=>{
       const {container}=  render(<NoRooms />)
        expect(container.firstChild).toMatchSnapshot();
      })
    test('should render the text',()=>{
        render(<NoRooms/>);
        screen.getByText(/No rooms at the moment/)
    })

})