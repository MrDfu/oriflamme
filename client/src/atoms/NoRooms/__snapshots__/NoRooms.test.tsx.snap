// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoRooms component should match the snapshot 1`] = `
<div
  class="no-rooms"
>
  No rooms at the moment
</div>
`;
